tests/Collection/DropEncryptedCollectionFunctionalTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace MongoDB\Tests\Collection;
+
+use MongoDB\BSON\Binary;
+use MongoDB\Database;
+use MongoDB\Driver\ClientEncryption;
+use MongoDB\Driver\WriteConcern;
+
+use function iterator_count;
+use function str_repeat;
+
+class DropEncryptedCollectionFunctionalTest extends FunctionalTestCase
+{
+    protected ClientEncryption $clientEncryption;
+    protected Database $database;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        $this->skipIfClientSideEncryptionIsNotSupported();
+
+        if ($this->isStandalone()) {
+            $this->markTestSkipped('Queryable encryption requires replica sets');
+        }
+
+        $this->skipIfServerVersion('<', '7.0.0', 'Queryable encryption requires MongoDB 7.0 or later');
+
+        $client = static::createTestClient();
+
+        // Ensure the key vault collection is dropped before each test
+        $collection = $client->selectCollection('keyvault', 'datakeys', ['writeConcern' => new WriteConcern(WriteConcern::MAJORITY)]);
+        $collection->drop();
+
+        $this->clientEncryption = $client->createClientEncryption([
+            'keyVaultNamespace' => 'keyvault.datakeys',
```

Comment:
To prevent any typo:
```suggestion
            'keyVaultNamespace' => $collection->getNamespace(),
```

Otherwise you don't really need the `$collection` variable.